### DISCUSSION TITLE
@types/sass -> RenderOption -> missing quietDeps

### DISCUSSION THREAD

**Discussion by meriturva** [DefinitelyTyped/DefinitelyTyped#54138] [Issues with a @types package]
See options introduced on sass 1.34:

https://github.com/sass/dart-sass/blob/7e371666f4f4554bb4e78de3087fec8177074b29/lib/src/node/render_options.dart#L29

`quietDeps` is missing on type definition for Options class:

```
export function render(options: Options, callback: (exception: SassException, result: Result) => void): void;
export function renderSync(options: Options): Result;
```

I guess maintainers are: @lenovouser and @jleider
is it possible to have an update with included flag `quietDeps`?

**Comment by jleider** [comment]
@meriturva I'm just a guy who wrote some types, feel free to submit a PR with the updated options.
